kratos/benchmarks/parallel_utilities_benchmark.cpp

```diff
@@ -48,7 +48,7 @@ static void BM_VectorPower(benchmark::State& state) {
     std::vector<double> data_vector(nsize, 5.0);
 
     for (auto _ : state) {
-        std::for_each(data_vector.begin(), data_vector.end(), [](double& item) {
+        block_for_each(data_vector, [](double& item) {
```

Comment:
you are changing this line from serial to parallel no?

Reply:
I was not testing the paralell utilities, I was using the std by mistake